WcaOnRails/app/models/team_member.rb

```diff
@@ -12,7 +12,7 @@ def current_member?
 
   validate :start_date_must_be_earlier_than_end_date
   def start_date_must_be_earlier_than_end_date
-    if end_date != nil && start_date >= end_date
+    if start_date && end_date && start_date >= end_date
```

Comment:
We have another validation for start_date presence below. So if it's empty the object is still invalid, but we disable this validation that would throw an exception.